components/style/properties/longhand/svg.mako.rs

```diff
@@ -158,3 +158,27 @@ ${helpers.predefined_type("mask-image", "ImageLayer",
     animation_value_type="discrete",
     flags="CREATES_STACKING_CONTEXT",
     has_uncacheable_values="True" if product == "gecko" else "False")}
+
+${helpers.predefined_type(
```

Comment:
Are these actual CSS properties? This will allow them to be parsed from style attributes, which may not be what you want.